src/Controller/ContactController.php

```diff
@@ -18,6 +18,9 @@
 class ContactController extends AbstractController
 {
     /**
+     * Recovery of the "ContactType" form to make the view.
+     * Checks the validity of the sent form and redirects to a
+     * page to confirm it was sent.
```

Comment:
Comments, specially above a controller method, should explain the purpose of the controller instead of rewording how it's done.
May be something like : "Visitors can send mail to ask information to the site admin using this form"